web_development_101/join_the_odin_community.md

```diff
@@ -65,9 +69,9 @@ For Multiple lines of code:
 Your Code
 #three backticks at the end on a separate line BELOW your code
 ```
-(make sure you are in **compose mode** by pressing ctrl + /, otherwise Enter will send instead of creating a new line)
+(for chat: make sure you are in **compose mode** by pressing ctrl + /, otherwise Enter will send instead of creating a new line)
```

Comment:
is this needed anymore? I've always told people to type three backticks, then hit enter and paste code between